tests/mq/plugins/test_suricataFixup.py

```diff
@@ -115,7 +115,8 @@ def test_suricata_notype_log(self):
         assert result['category'] == 'suricata'
```

Comment:
This file has a lot of assertions for doc_type that are not addressed. Some are doc_type = 'event' some are doc_type = 'nsm'  and they are not on the tests that check for doc_type being incorrect.

Reply:
Based off our conversation off line, I think we're good here (and these will get officially cleaned up when we remove doc_type entirely due to ES6)